volatility3/framework/objects/__init__.py

```diff
@@ -602,6 +602,14 @@ def relative_child_offset(cls, template: interfaces.objects.Template, child: str
                 return 0
             raise IndexError(f"Member not present in array template: {child}")
 
+        @classmethod
+        def child_template(cls, template: interfaces.objects.Template, child: str) -> interfaces.objects.Template:
+            """Returns the template of the child member."""
+            if 'subtype' in template.vol and child == 'subtype'@
```

Comment:
It seems typo of if statement syntax.

```suggestion
            if 'subtype' in template.vol and child == 'subtype':
```

Reply:
You're quite right, thanks for the spot!  5:)